tests/test.sh

```diff
@@ -121,6 +121,22 @@ fi
 
 check_valgrind_log "bad words"
 
+# Tests good words' root
+if test -f $TESTDIR/$NAME.root; then
+    hunspell $* -d $TESTDIR/$NAME <$TESTDIR/$NAME.good | grep -a '^+ ' | \
```

Comment:
OK. It extracts root words (on '+ ' lines) of the prefixed or suffixed words. There was no test code for checking the root words so I had to add one.